### PR TITLE
Removed success flash on get next, fixed style and layout on your list

### PR DESCRIPTION
**Pull request by arthurashman**

### CODE REVIEW

File: app/views/assigned_applications/index.html.erb [hunk 1]
```diff
@@ -6,44 +6,52 @@
...
+
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full govuk-!-margin-bottom-5">
-    <%= button_to(
-          'Get next application',
-          get_next_assigned_applications_path,
-          method: :post,
-          class: 'govuk-button'
-        ) %>
+    <%= form_with url: get_next_assigned_applications_path, method: :post do |f| %>
```
Comment (willmcb) [inline app/views/assigned_applications/index.html.erb:12]: Forms default to post so I don't think we need to specify it.

File: app/controllers/assigned_applications_controller.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@ def create
     )
   end
 
-  # rubocop:disable Naming/AccessorMethodName, Metrics/MethodLength, Metrics/AbcSize
+  # rubocop:disable Naming/AccessorMethodName, Metrics/MethodLength
```
Comment (willmcb) [inline app/controllers/assigned_applications_controller.rb:21]: I think we could remove both of these opt outs:

1. Call it perhaps `next_application`
2. collapse some of these lines they aren't that long that it should matter

File: app/views/assigned_applications/index.html.erb (hunk 1 above)
Comment (willmcb) [reply app/views/assigned_applications/index.html.erb:12]: Just thinking as well why not just keep the button with the post? there is only one element in this form so its functially in the same. I think we could have the link outside the form as it is not using the form object and then just wrap them in a row.

File: app/views/assigned_applications/index.html.erb [hunk 3]
```diff
@@ -6,44 +6,52 @@
...
-        </tr>
-      </thead>
-      <tbody class="govuk-table__body">
-        <% @applications.each do |app| %>
-          <%= render 'application/crime_application', crime_application: app %>
-        <% end %>
-      </tbody>
-    </table>
+<% if @applications.any? %>
+  <div class="govuk-grid-row">
```
Comment (willmcb) [inline app/views/assigned_applications/index.html.erb:27]: Wondering if we could wrap this table in a partial - might make this view a bit easier to parse.

File: config/locales/en.yml [hunk 4]
```diff
@@ -35,7 +35,7 @@ en:
     open_apps: All open applications
 
   labels: &LABELS
-    assigned_to: Assigned to
+    assigned_to: 'Assigned to:'
```
Comment (willmcb) [inline config/locales/en.yml:38]: Do we need apostrophes here? I think remove them unless we do for some reason

**Review by willmcb — COMMENTED** [review]
Few points to consider here :)

File: config/locales/en.yml (hunk 4 above)
Comment (arthurashman) [reply config/locales/en.yml:38]: The colon makes the yml think it's another key, so need to be in quotes. Other option apparently to do:
key: >
  Value that has: colon
  
But the quotes seems cleaner

File: app/controllers/assigned_applications_controller.rb (hunk 2 above)
Comment (arthurashman) [reply app/controllers/assigned_applications_controller.rb:21]: 👍

**Review by willmcb — APPROVED** [review]
LGTM :)

File: app/views/assigned_applications/index.html.erb (hunk 3 above)
Comment (arthurashman) [reply app/views/assigned_applications/index.html.erb]: Done

**Pull request merged**